### ISSUE TITLE
Refactor: move LiveStore from solid-logic to rdflib

### ISSUE THREAD

**Issue by theRealImy** [solid/solidos#109]

**Comment by theRealImy** [comment]
This refactoring spans over a lot of repos. 
SolidOS PR: https://github.com/solid/solidos/pull/110
The modules depend on each other so they have to be built in the right order and with the right rdflib commit version ( "rdflib": "^2.2.15-bbf45639")

- [ ] solid-logic: https://github.com/solid/solid-logic/pull/26
- [ ] pane-registry: https://github.com/solid/pane-registry/pull/17
- [ ] solid-ui: https://github.com/solid/solid-ui/pull/465
- [ ] contacts-pane: https://github.com/solid/contacts-pane/pull/61
- [ ] profile-pane: https://github.com/solid/profile-pane/pull/27
- [ ] issue-pane: https://github.com/solid/issue-pane/pull/51
- [ ] meeting-pane: https://github.com/solid/meeting-pane/pull/43
- [ ] activitystream-pane: https://github.com/solid/activitystreams-pane/pull/8
- [ ] solid-panes: https://github.com/solid/solid-panes/pull/332

I hope I got all dependencies that needed the change. Needs testing.

**Comment by bourgeoa** [comment]
I think chat-pane is missing.

**Comment by theRealImy** [comment]
> I think chat-pane is missing.

@bourgeoa chat-pane only depends on solid-ui and it does not use LiveStore. At least I did not find it. This is teh reason I did not change anything in it and it is not in the list.

**Comment by bourgeoa** [comment]
Using published solid-ui@2.4.14-bbf45639
from branch move-logic with :
```
    "pane-registry": "^2.4.5-bbf45639",
    "rdflib": "^2.2.15-bbf45639",
    "solid-logic": "^1.3.12-bbf45639",
```
All passes lint, tsc, test, build with all prior packages in.

**Comment by theRealImy** [comment]
All done, tested and merged to main.
